File: MicroSocialServer/MicroSocialServer/Schema/Status.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

using Newtonsoft.Json;

namespace MicroSocialServer.Schema
{
    class Status
    {
        public string poster;
        public DateTime time;
        public string statusContent;

        public Status()
        {

        }
    }
}

File: MicroSocialAndroid/app/src/main/java/me/omegavesko/microsocial/android/alpha/schema/Status.java
package me.omegavesko.microsocial.android.alpha.schema;

import java.util.Date;

public class Status
{
    public User poster;
    public String time; // DateTime string
    public String statusContent;

    public Status()
    {

    }

    public Status(User poster, String time, String statusContent)
    {
        this.poster = poster;
        this.time = time;
        this.statusContent = statusContent;
    }
}